$\textbf{Empirical Exercise-Week 4}$  Page 245, Problem 5.28
Let's look at an example about the auction market for art. 

In [1]:
clear all

Tab1 = readtable('ashcan_small.csv');  sold = Tab1(:,1);  

In [2]:
Tab1(1,:)
who


ans =

  1x10 table

    sold    date_auctn    years_old    rhammer    inchsq    signed    creation    age    yob     drec
    ____    __________    _________    _______    ______    ______    ________    ___    ____    ____

     1        2008.8       104.75      0.17327    81.04       1         1904      39     1865     1  


Your variables are:

Tab1  ans   sold  




In [3]:
 rhammer = Tab1(:,4);  inchsq = Tab1(:,5); yrsold = Tab1(:,3);  

In [4]:
rhammer1 = table2array(rhammer); inchsq1 = table2array(inchsq); yrsold1 = table2array(yrsold); 
sold1 = table2array(sold);
dummy = (sold1== 1); %  only use data for iterms which are sold
rhammer2 = rhammer1(dummy,:); inchsq2 = inchsq1(dummy,:)/10; yrsold2 = yrsold1(dummy,:);

In [5]:
y = log(rhammer2);  xxx = [ones(length(y),1) yrsold2 inchsq2];

In [6]:
[B, BINT, R, STATS]= regress(y,xxx);
BINT


BINT =

   -1.1663    0.6762
    0.0110    0.0322
    0.0213    0.0321




Answer to question: how much paintings appreciate in one year, between 1.1% and 3.22% at the 95 percent level of confidence

With a 95% confidence interval we can also say that an extra 10 square inches will appreciate a painting betweeh 2.13% and 3.21%. We can reject the null hypothesis that an extra 10 inches will appreciate a painting by 2% in favor of the one-sided alternative that the appreciation will be greater than 2%.

In [7]:
xxxx = [ones(length(y),1) yrsold2 inchsq2 inchsq2 .^2];   % Created quadratic term

In [8]:
[B1 BINT1] = regress(y,xxxx);
BINT1


BINT1 =

   -1.8628   -0.0522
    0.0128    0.0331
    0.0459    0.0665
   -0.0002   -0.0001




The addition of the quadratic term does not change the confidence interval for age.

In [9]:
median(inchsq2)


ans =

   25.4062




Confidence interval for changing by 50, 250, 900 square inches

In [10]:
[BINT1(2,:)*5 ; BINT1(2,:)*25; BINT1(2,:)*90]


ans =

    0.0641    0.1655
    0.3206    0.8277
    1.1543    2.9797




Confidence interval for price-maximizing size of painting: $inchsq2^{*} = .5 * beta2/beta3$

In [11]:
-[.5*BINT1(3,1)/BINT1(4,1)  .5*BINT1(3,2)/BINT1(4,2)]


ans =

   94.8419  257.0542




Answer is the expected log price and the 95% confidence level. Remember that a negative log value is still a positive price.

In [13]:
save Exercise_Week4;  clear all